attributes/nginx.rb

```diff
@@ -15,6 +15,7 @@
 
 default['nginx']['port'] = node['alfresco']['public_port']
 default['nginx']['portssl'] = node['alfresco']['public_portssl']
+default['nginx']['status_port'] = 2100
```

Comment:
I suppose we need to open this port on iptables (or our Vagrantfile.erb, so that it can be mapped in the host machine)